docs/website/docs/guides/deployment-configurations/gpu-vulkan.md

```diff
@@ -193,22 +193,22 @@ iree-compile \
 !!! note
     Currently a target triple of the form `<vendor/arch>-<product>-<os>` is needed
     to compile towards a specific GPU architecture.
-    
+
     We don't support the full spectrum here[^1]; the following table summarizes the
```

Comment:
The lint error here can either be fixed with
```
<!-- markdownlint-disable-next-line -->
```
or with this patch: https://github.com/ScottTodd/iree/commit/697ef3cf526d72be58a77681242a3e770534b0ea

That patch uses an [annotation](https://squidfunk.github.io/mkdocs-material/reference/annotations/) instead of a [footnote](https://squidfunk.github.io/mkdocs-material/reference/footnotes/): 
![image](https://github.com/openxla/iree/assets/4010439/30627ee1-f6bc-40e4-9250-8409360b8ecb)

I don't have a strong preference for annotation vs footnote here.

Reply:
The annotation looks more straightforward to me. I merged the commit into my branch, thanks for the quick fix!